crates/burn-import/src/burn/node/binary.rs

```diff
@@ -14,6 +14,8 @@ pub enum BinaryType {
     Equal,
     Powf,
     Powi,
+    FloatCumsum,
+    IntCumsum,
```

Comment:
[`CumSum`](https://github.com/onnx/onnx/blob/main/docs/Operators.md#Cumsum) is a single operator, the same for int and float.

So we should also only have one node type, `BinaryType::Cumsum`. See for example `BinaryType::Sub`, which does split the int and float tests for the generated onnx files but it's still a single operation/node.